cmd/bench.go

```diff
@@ -99,7 +100,7 @@ The optional "gobench" output format conforms to the Go Benchmark Data Format.
 }
 
 type benchRunner interface {
-	run(ctx context.Context, ectx *evalContext, pq rego.PreparedEvalQuery, params benchmarkCommandParams) (testing.BenchmarkResult, error)
+	run(ctx context.Context, ectx *evalContext, params benchmarkCommandParams, f func(context.Context, ...rego.EvalOption) error) (testing.BenchmarkResult, error)
```

Comment:
[nit] we could name the function type, like `type benchFunc func(context.Context, ...rego.EvalOption) error`. But it has little impact outside of the use in `bench_test.go` below.